jdbc/src/java/org/apache/hive/jdbc/HiveQueryResultSet.java

```diff
@@ -290,7 +328,7 @@ public void close() throws SQLException {
     client = null;
     stmtHandle = null;
     isClosed = true;
-    operationStatus = null;
+    InitEmptyIterator();
```

Comment:
`camel-case`

Reply:
Have to restore initial state in case result is re-opened.